libs/sdk-ui-tests-e2e/package.json

```diff
@@ -32,38 +32,46 @@
         "styles"
     ],
     "config": {
-        "eslint": "-c .eslintrc.js --ext ts,tsx cypress/"
+        "eslint": "-c .eslintrc.cjs --ext js,jsx,ts,tsx ./"
     },
     "scripts": {
-        "clean": "rm -rf ci dist esm coverage *.log styles/css",
+        "clean": "rm -rf ci dist esm scenarios/build coverage *.log styles/css",
         "api-extractor": "mkdir -p api && [ -z \"${CI}\" ] && (api-extractor run -l) || (api-extractor run)",
         "build": "tsc -p tsconfig.build.json",
         "dev": "bash scripts/build.sh --dev-watch",
         "eslint": "eslint $npm_package_config_eslint",
         "eslint-ci": "mkdir -p ./ci/results && eslint -f checkstyle -o ci/results/eslint-results.xml $npm_package_config_eslint",
-        "prettier-check": "prettier --check '{src,stories,styles,__mocks__}/**/*.{ts,tsx,json,scss,md,yaml,html}'",
-        "prettier-write": "prettier --write '{src,stories,styles,__mocks__}/**/*.{ts,tsx,json,scss,md,yaml,html}'",
-        "dep-cruiser": "depcruise --validate .dependency-cruiser.js --output-type err-long cypress/",
-        "dep-cruiser-ci": "depcruise --validate .dependency-cruiser.js --output-type err-long cypress/",
-        "validate": "npm run dep-cruiser && npm run eslint && npm run prettier-check",
-        "validate-ci": "npm run dep-cruiser-ci && npm run eslint-ci && npm run prettier-check",
-        "cypress": "node ./scripts/run_cypress_local.js",
-        "docker-build": "npm run build-scenarios && DOCKER_BUILDKIT=0 docker build --file Dockerfile -t gooddata-ui-sdk .",
+        "prettier-check": "prettier --check './**/*.{ts,js,cjs,tsx,jsx,json,scss,md,yaml,html}'",
+        "prettier-write": "prettier --write './**/*.{ts,js,cjs,tsx,jsx,json,scss,md,yaml,html}'",
+        "dep-cruiser": "depcruise --validate .dependency-cruiser.cjs --output-type err-long cypress/",
+        "dep-cruiser-ci": "depcruise --validate .dependency-cruiser.cjs --output-type err-long cypress/",
+        "validate": "yarn dep-cruiser && yarn eslint && yarn prettier-check",
+        "validate-ci": "yarn dep-cruiser-ci && yarn eslint-ci && yarn prettier-check",
```

Comment:
Why did you change the `npm run XXX` to `yarn XXX`? We do not use `yarn` in the SDK repo at all. If it is not absolutely necesary please, use `npm run`

Reply:
fixed